include/openthread-ip6.h

```diff
@@ -0,0 +1,507 @@
+/*
```

Comment:
Would it be better to separate out the message, IPv6 and UDP APIs into `openthread-message.h`, `openthread-ip6.h`, and `openthread-udp.h`, respectively?